### PR TITLE
prop paginatorLabels in gridConfig options

### PR DESCRIPTION
**Pull request by thotty**
Now we can pass to grid config the `paginatorLabels` object with this (key: 'value default'):
-  `stepBack: '‹'`
-  `stepAhead: '›'`
-  `jumpBack: '«'`
-  `jumpAhead: '»'`
-  `first: 'First'`
-  `last: 'Last`

### CODE REVIEW

**Comment by thotty** [comment]
I can't commited the **angular-table.js** and **angular-table.min.js** in **dist** folder… Ok?

File: coffee/configuration/scope_config_wrapper.coffee [hunk 1]
```diff
@@ -18,3 +18,13 @@ class ScopeConfigWrapper
 
   setCurrentPage: (currentPage) ->
     @scope.$eval("#{@configurationVariableNames.currentPage}=#{currentPage}")
+
+  getpaginatorLabels: () ->
```
Comment (samu) [inline coffee/configuration/scope_config_wrapper.coffee]: Please use correct camel case: `getPaginatorLabels`

**Comment by samu** [comment]
Hi @thotty, thanks for the PR.

There seems to be a merge conflict, please resolve it so i can pull.

Also, please note the comment above about proper camel case naming.

Thanks :)

**Comment by thotty** [comment]
Oh, man.. My bad! I used a "replace all" functionality and not saw this..
I will correct to send again :smile:

**Comment by thotty** [comment]
@samu, now is ready to pull... You agree?

**Comment by samu** [comment]
@thotty Thanks, great work :smile:

**Comment by thotty** [comment]
Thank you, @samu.

**Pull request merged**